packages/clay-management-toolbar/src/SearchInput.tsx

```diff
@@ -14,8 +14,14 @@ interface IProps {
 	onCloseButtonClick: (
 		event: React.MouseEvent<HTMLButtonElement, MouseEvent>
 	) => void;
-	onlySearch?: boolean;
+	onSubmit: (
+		event:
```

Comment:
Hmm, in DXP you may not want to javascript control the request, since the form here is wrapped with a `form` tag. We had cases like this when the component was being used with Taglib, but I think we should support it here too.

I may have to add a new `searchFormProps` prop here to move to` SearchForm`.

Reply:
Oh thats true. I didn't realize the button had `type="submit"`. For clarity then, would you be alright with calling it `onValueSubmit` and then changing the order of the args `onSubmit(value, event)`?
- By changing the name to `onValueSubmit` I think he help avoid confusion with HTMLs `onSubmit` event which wouldn't make sense with the click event.
- By changing the order of the args, I think it prioritizes the value rather than the event. Which I think value will be used more than event. I don't have proof of that, its just an assumption lol

Reply:
Hmm, if we were to follow the `onSubmit` specification we would just have to pass the event and dev would have to get the value through the event.

But I think we can go with `onValueSubmit`.